lib/course_planner/scheduler.ex

```diff
@@ -0,0 +1,5 @@
+defmodule CoursePlanner.Scheduler do
```

Comment:
we already have a module name ScheduleController and ScheduleView. what do you think of making this one a bit more explicit. something like "email_scheduler" or etc